### PR TITLE
Check if request is handled before continue() or abort() is called

### PR DESCRIPTION
**Pull request by ItzRabbs**
In reference to issue https://github.com/berstend/puppeteer-extra/issues/156

### CODE REVIEW

File: packages/puppeteer-extra-plugin-block-resources/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppeteer-extra-plugin-block-resources",
-  "version": "2.2.4",
+  "version": "2.2.5",
```
Comment (brunogaspar) [inline packages/puppeteer-extra-plugin-block-resources/package.json:3]: Can you revert this change please?

File: packages/puppeteer-extra-plugin-block-resources/index.js [hunk 2]
```diff
@@ -111,6 +112,6 @@ class Plugin extends PuppeteerExtraPlugin {
   }
 }
 
-module.exports = function(pluginConfig) {
+module.exports = function (pluginConfig) {
```
Comment (brunogaspar) [inline packages/puppeteer-extra-plugin-block-resources/index.js:115]: Can you revert this change please?

There's no strict guidelines on how the code should be linted, therefore it's best to keep it as similar as possible for now :)

**Comment by berstend** [comment]
This PR is one year old, closing in favor of the discussions happening at #156 and #364
